recipes/pytest-asyncio-cooperative/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pytest-asyncio-cooperative" %}
+{% set version = "0.28.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-asyncio-cooperative-{{ version }}.tar.gz
+  sha256: 1085a3d92b575315652cf05493da243bb620622e06a83096488afc6d250f1b9a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - poetry >=1.3.1
+  run:
+    - python >=3.7
```

Comment:
Upstream data doesn't specify but I believe this requires pytest to run, no?

Reply:
it does as the name suggests :), I added the dep below :+1: